SpatialGDK/Source/SpatialGDKTests/Private/RPCContainer/ObjectSpy.h

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Utils/RPCContainer.h"
+
+#include "CoreMinimal.h"
+
+#include "ObjectSpy.generated.h"
+
+namespace SpyUtils
+{
+	TArray<uint8> TypeToArray(ESchemaComponentType Type);
+	ESchemaComponentType ArrayToType(const TArray<uint8>& Array);
+}
```

Comment:
comment note at the end of the namespace

Reply:
Updated.